source/gsoc/ideas2015.rst

```diff
@@ -0,0 +1,137 @@
+Google Summer of Code 2015 Ideas
+================================
+
+Welcome to the Oregon State University Open Source Lab Google Summer of Code
+2015 idea page! Before applying please read over our :ref:`application guide
+<gsoc-app-info>` for details specific to our organization.
+
+Ideas
+-----
+- :ref:`gsoc2015-data-center-tools`
+- :ref:`gsoc2015-whats-fresh`
+- :ref:`gsoc2014-other-projects`
+
+.. _gsoc2015-data-center-tools:
+
+Data Center Tools
+~~~~~~~~~~~~~~~~~
+
+Project: Infrastructure-wide IPMI/SNMP Management Console
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Brief Explanation:**
+
+We host a variety of systems in our data center which include a wide array of
+vendors (i.e. Dell, HP, Sun, etc). Each of these machines utilize some type of
+out-of-band (OOB) technology such as iLO or iDrac which allows direct access to
+the machine power, console and other features. Unfortunately each OOB tool is
+different between the command line access and the console access. 
+
+However server machines support the IPMI standard which basically does the same,
+and PDUs (Power Distribution Units) use another common standard, SNMP.
+
+There are several FOSS tools out there already that utilize IPMI and SNMP in some
+fashion, but none create a complete tool that stores a whole infrastructure into
+an interface. We have begun work on an IPMI management tool, and would like to 
+integrate SNMP functionality with this to create a unified OOB management tool.
+
+**Expected Results**
+
+Either a command line driven or web app driven interface that allows for
+viewing and addition of machines and PDUs, access to power, serial and vga console,
+authentication of users and authorization by user and group. Optionally show event 
+logs, time-based metrics, and any sensor data.
+
+**Knowledge Prerequisite:**
+
+Basic understanding of production-grade datacenter hardware, ability to program in
+Python, and a good grasp of API driven interfaces.
+
+**Mentor:** Lance (Ramereth)
+
+------
+
+.. _gsoc2015-whats-fresh:
+
+What's Fresh
+~~~~~~~~~~~~
+
+Project: What's Fresh version 2.0
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What's Fresh is an open source project originally funded by the Oregon Sea Grant
```

Comment:
You should add a link to the github repo and maybe a link to Oregon Sea Grant's site.